src/MMKV.ts

```diff
@@ -46,6 +54,20 @@ export interface MMKVConfiguration {
    * ```
    */
   encryptionKey?: string;
+  /**
+   * *Android Only*: The MMKV mode. It is set to SingleProcess by default. You can set its value to MultiProcess to support simultaneous read-write access between processus at the cost of performance.
+   *
+   * This is useful when you want to share data between your react-native app and native extensions such as widgets.
+   *
+   * @example
+   * ```ts
+   * const extensionStorage = new MMKV({ mode: MMKVMode.MultiProcess })
+   * ```
+   *
+   * _Notice_: On iOS, this will automatically be set to MMKVMode.MultiProcess if you set an AppGroup in the plist configuration.
+   * More information on AppGroups [here](https://github.com/mrousavy/react-native-mmkv/tree/master#app-groups)
+   */
+  mode?: MMKVMode;
```

Comment:
Can you please use TypeScript unions instead of enums here?

So instead of `mode?: MMKVMode` it would be `mode?: 'single-process' | 'multi-process'`